packages/graph/hash_graph/lib/graph/src/ontology/data_type.rs

```diff
@@ -11,24 +11,149 @@ use crate::store::query::{OntologyPath, ParameterType, QueryRecord, RecordPath};
 /// A path to a [`DataType`] field.
 ///
 /// Note: [`DataType`]s currently don't reference other [`DataType`]s, so the path can only be a
-/// single field.
+/// single field. This means, that the path currently will always be a sequence with only one
+/// element.
 ///
 /// [`DataType`]: type_system::DataType
 // TODO: Adjust enum and docs when adding non-primitive data types
 //   see https://app.asana.com/0/1200211978612931/1202464168422955/f
 #[derive(Debug, PartialEq, Eq)]
 pub enum DataTypeQueryPath {
-    VersionId,
+    /// The [`BaseUri`] of a [`DataType`].
+    ///
+    /// ```rust
+    /// # use serde::Deserialize;
+    /// # use serde_json::json;
+    /// # use graph::ontology::DataTypeQueryPath;
+    /// let path = DataTypeQueryPath::deserialize(json!(["baseUri"]))?;
+    /// assert_eq!(path, DataTypeQueryPath::BaseUri);
+    /// # Ok::<(), serde_json::Error>(())
+    /// ```
+    ///
+    /// [`BaseUri`]: type_system::uri::BaseUri
+    BaseUri,
+    /// The version of the [`DataType`].
+    ///
+    /// ```rust
+    /// # use serde::Deserialize;
+    /// # use serde_json::json;
+    /// # use graph::ontology::DataTypeQueryPath;
+    /// let path = DataTypeQueryPath::deserialize(json!(["version"]))?;
+    /// assert_eq!(path, DataTypeQueryPath::Version);
+    /// # Ok::<(), serde_json::Error>(())
+    /// ```
+    ///
+    /// In addition to specifying the version directly, it's also possible to compare the version
+    /// with a `"latest"` parameter, which will only match the latest version of one
```

Comment:
Should we use each instead? 
```suggestion
    /// with a `"latest"` parameter, which will only match the latest version of each
```

Reply:
I almost copy/pasted this snippet for `EntityQueryPath` and noticed the "one" and replaced it. Next suggestion from you is to change "one", amazing 😆 🎉